files/paths_stats.md

```diff
@@ -0,0 +1,52 @@
+## Path Performance Information
+### A path is a collection of routes that serve a common corridor or origin-destination pair
+
+ *  Filename MUST be `paths.txt`
```

Comment:
This should probably be `***_stats.txt` ?

Reply:
oops.